code/modules/urist/random.dm

```diff
@@ -1,10 +1,5 @@
 //stuff that doesn't go anywhere else.
 
-/obj/effect/stop/Uncross(atom/movable/O)
-	if(victim == O)
-		return 0
-	return 1
-
```

Comment:
i changed this to /obj/urist_intangible/stop/Uncross(atom/movable/O) locally, but is this even used anywhere?

Reply:
Yeah, I haven't seen it - hence the removal. I think it may be a holdover from 2d shuttle world?

Uncross is an engine-level method, messing with it risks breaking Baycode stuff, so we need to make sure we know why they're there.